website/docs/release_notes.md

```diff
@@ -5,6 +5,97 @@ sidebar_position: 99
 
 # Release notes
 
+## [v0.1.11](https://github.com/VirtusLab/scala-cli/releases/tag/v0.1.11)
+
+### Add support for running multiple snippets of the same kind
+Scala CLI now allows passing and running multiple snippets. Each snippet is then treated as a
+separate input by Scala CLI.
+
+```bash
+scala-cli --scala-snippet '@main def main() = println(Messages.hello)' --scala-snippet 'object Messages { def hello = "Hello" }'
+```
+
+```text
+Hello
+```
+Added in [#1182](https://github.com/VirtusLab/scala-cli/pull/1182) by [Gedochao](https://github.com/Gedochao)
+
+### Make main class optional in preamble-less assemblies
+It is now allowed to generate an assembly, even for code that has no main class, when `--preamble=false` is passed. This can be useful for libraries, if users want to pass the assembly to tools such as proguard. This also accepts a (hidden) `--main-class-in-manifest=false` option if users want not only no preamble, but also no mention of main class in the assembly manifest (`META-INF/MANIFEST.MF` in the assembly JAR). The latter option is useful for tools, such as the hadoop jar command, that behave differently depending on the presence or not of a main class in the manifest.
+
+Added in [#1200](https://github.com/VirtusLab/scala-cli/pull/1200) by [alexarchambault](https://github.com/alexarchambault)
+
+### Generate scalafmt conf file on fmt command
+Scala CLI now generates a default `.scalafmt.conf` file using `fmt`, `format` or `scalafmt` command if such file is absent. Previously, a `Scalafmt requires explicitly specified version.` error would be raised instead.
+
+Added in [#1192](https://github.com/VirtusLab/scala-cli/pull/1192) by [wleczny](https://github.com/wleczny)
+
+### Add bloop sub-command
+Scala CLI now has a (hidden for now) bloop sub-command, that runs a command using the Scala CLI Bloop server (while the mainline Bloop bloop CLI uses its default Bloop server). This is handy when debugging things on Scala CLI for example, allowing one to manually run scala-cli bloop projects or scala-cli bloop compile.
+
+Added in [#1199](https://github.com/VirtusLab/scala-cli/pull/1199) by [alexarchambault](https://github.com/alexarchambault)
+
+### Define `output` option for `package` command with using directives
+It is now possible to pass some of the `package` options with [using directives](/docs/guides/using-directives) instead of passing them directly from bash.
+
+Added in [#1213](https://github.com/VirtusLab/scala-cli/pull/1213) by [wleczny](https://github.com/wleczny)
+
+### Important fixes & enhancements
+#### Prevent erroneous using directives from blocking the initial run of BSP
+Erroneous `using` directives are now prevented from blocking the initial BSP setup (fixes [#1097](https://github.com/VirtusLab/scala-cli/issues/1097))
```

Comment:
I think this fix can be described in more detail.
```suggestion
Up till now, running the `setup-ide` sub-command on sources containing `using directives` with syntax errors or pointing to dependencies which could not be fetched would create a `BSP` setup which could not be imported correctly by IDEs. This is no longer the case and such a `BSP` connection should now import correctly, so that it's possible to fix the faulty code within the comfort of one's IDE of choice.

This fixes [#1097](https://github.com/VirtusLab/scala-cli/issues/1097)
```

Reply:
Changed